src/hotspot/share/gc/shenandoah/heuristics/shenandoahHeuristics.hpp

```diff
@@ -78,6 +78,10 @@ class ShenandoahHeuristics : public CHeapObj<mtGC> {
 protected:
   static const uint Moving_Average_Samples = 10; // Number of samples to store in moving averages
 
+  bool _start_gc_is_pending;
```

Comment:
A description of each of these members and how they operate in this new protocol would be helpful.

Reply:
I've added descriptions.  Thanks.